src/main/app/case/answers/getAnswerRows.ts

```diff
@@ -27,16 +28,15 @@ export const getAnswerRows = function (
     formState: Partial<Case>;
   } = this.ctx;
 
-  let stepsWithContent = isApplicant2 ? stepsWithContentApplicant2 : stepsWithContentApplicant1;
-  if (overrideStepsContent === 1) {
-    stepsWithContent = stepsWithContentApplicant1;
-  } else if (overrideStepsContent === 2) {
-    stepsWithContent = stepsWithContentApplicant2;
-  }
-  const processedFormState = omitUnreachableAnswers(formState, stepsWithContent);
+  const { stepsWithContent, processedFormState } = setUpSteps(
+    formState,
+    isCompleteCase,
+    overrideStepsContent,
+    isApplicant2
+  );
 
   return stepsWithContent
-    .filter(step => step.showInSection === section)
+    .filter(step => (isCompleteCase ? step.showInCompleteSection === section : step.showInSection === section))
```

Comment:
as alot of these are duplicates, maybe we can do
```suggestion
    .filter(step => (isCompleteCase && step.showInCompleteSection ? step.showInCompleteSection === section : step.showInSection === section))
```
that way we dont need to duplicate it in the sequence.

Reply:
@velimd This doesn't work because if it is a Complete section, and we don't want a specific step to show up, but the step is assigned a 'showInSection' it will show up